tgui/packages/tgui/interfaces/ReagentGrinder.js

```diff
@@ -0,0 +1,166 @@
+  const { data } = useBackend(context);
+  const {
+          operating,
+          name
+        } = data;
+
+  if (operating) {
+    return (
+      <Dimmer>
+        <Flex mb="30px">
+          <Flex.Item
+            bold
+            color="silver"
+            textAlign="center">
+            <Icon
+              name="spinner"
+              spin
+              size={4}
+              mb="15px"
+            /><br />
+            The {name} is processing...
+          </Flex.Item>
+        </Flex>
+      </Dimmer>
+    );
+  }
+};
+
+const GrinderControls = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+          inactive,
+        } = data;
+
+  return (
+    <Section title="Controls">
+      <Flex>
+        <Flex.Item width="50%" mr="3px">
+          <Button
+            fluid
+            textAlign="center"
+            icon="mortar-pestle"
+            disabled={inactive}
+            tooltip={inactive ? "There are no contents" : "Grind the contents"}
+            tooltipPosition="bottom"
+            content="Grind"
+            onClick={() => act('grind')}
+          />
+        </Flex.Item>
+        <Flex.Item width="50%">
+          <Button
+            fluid
+            textAlign="center"
+            icon="blender"
+            disabled={inactive}
+            tooltip={inactive ? "There are no contents" : "Juice the contents"}
+            tooltipPosition="bottom"
+            content="Juice"
+            onClick={() => act('juice')}
+          />
+        </Flex.Item>
+      </Flex>
+    </Section>
+  );
+}
+
+const GrinderContents = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    contents,
+    limit,
+    count,
+    inactive
+  } = data;
+
+  return (
+    <Section
+    title="Contents"
+    flexGrow={1}
+    buttons={(
+      <Box>
+        <Box inline color="label" mr={2}>
+          {count} / {limit} items
+        </Box>
+        <Button
+            icon="eject"
+            content="Eject Contents"
+            onClick={() => act('eject')}
+            disabled={inactive}
+            tooltip={inactive ? "There are no contents" : ""}
+          />
+      </Box>
+    )}
+    >
+      <Table>
+        {contents.map((content) => (
+          <Table.Row tr={5} key={content.name}>
+            <td>
+              <Table.Cell bold>{content.name}</Table.Cell>
+            </td>
+            <td>
+              <Table.Cell collapsing textAlign="center">
+                {content.amount} {content.units}
+              </Table.Cell>
+            </td>
+          </Table.Row>
+        ))}
+      </Table>
+    </Section>
+  )
+}
+
+const GrinderReagents = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+          beaker_loaded,
+          beaker_current_volume,
+          beaker_max_volume,
+          beaker_contents
+        } = data;
+
+  return (
+    <Section
+    title="Beaker"
+    flexGrow="1"
+    buttons={
```

Comment:
```suggestion
        title="Beaker"
        flexGrow="1"
        buttons={
```
These can be done just like Section above.